docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1647 @@
+{
```

Comment:
<p>In the different sections can we have a little bit of setup? Possibly with some linking to user-guide docs as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>